Docs/ProjectSpec.md

```diff
@@ -364,24 +364,25 @@ targets:
 
 ### Dependency
 
-A dependency can be one of a 3 types:
+A dependency can be one of a 5 types:
 
 - `target: name` - links to another target
 - `framework: path` - links to a framework
 - `carthage: name` - helper for linking to a Carthage framework
 - `sdk: name` - links to a dependency with the SDK. This can either be a relative path within the sdk root or a single filename that references a framework (.framework) or lib (.tbd)
+- `bundle: name` - adds the pre-built bundle for the supplied name to the copy resources build phase. This is useful when a dependency exists on a static library target that has an associated bundle target, both existing in a separate project. Only usable in targets types which can copy resources. Always use `implicit: true`. This will will not cause the bundle to be explicitly built.
```

Comment:
For better readability: Only usable in `target` types which ...
Typo: `This will will ...`

Reply:
👍

Reply:
What happens if someone doesn't put `implicit`? Should this be the implied default, so people don't have to set it?